pkg/machinery/config/types/v1alpha1/v1alpha1_provider.go

```diff
@@ -1384,6 +1384,16 @@ func (e *EncryptionKey) TPM() config.EncryptionKeyTPM {
 	return e.KeyTPM
 }
 
+// String implements the config.Provider interface.
+func (e *EncryptionKeyNodeID) String() string {
+	return "tpm"
```

Comment:
is returning `tpm` here intentional (and not `node-id`)?

Reply:
co-pilot :joy:

Reply:
happens to the best of us :D